x-pack/plugins/security_solution/cypress/integration/detection_rules/prebuilt_rules.spec.ts

```diff
@@ -46,7 +47,13 @@ describe('Prebuilt rules', () => {
 
   describe('Alerts rules, prebuilt rules', () => {
     it('Loads prebuilt rules', () => {
-      const rowsPerPage = 100;
+      cy.intercept('/api/detection_engine/rules/_find*', {
```

Comment:
@banderror Is this what you were thinking?

Here I had to mock the response of the `find rules` call, because a couple of calls are made before the rules are successfully installed, and the data returned is an empty array. By mocking the response I can assert that the response of the API call is used to populate the Rules table. The responsibility of ensuring that the API call itself works as expected is left [to the integration test.](/api/detection_engine/rules/_find)

What do you think?

Reply:
@jpdjere No, I don't think we should mock the response of this endpoint. We should make a call and use the real response to validate that all `rawRules` have been actually created.

Reply:
Updated to make actual requests and assert on its response.